cico_build_deploy.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+GENERATOR_DOCKER_HUB_USERNAME=openshiftioadmin
+REGISTRY_URI="push.registry.devshift.net"
+REGISTRY_NS="openshiftio"
+REGISTRY_IMAGE="appdev-documentation"
+REGISTRY_URL=${REGISTRY_URI}/${REGISTRY_NS}/${REGISTRY_IMAGE}
+DOCKER_HUB_URL="openshiftio/appdev-documentation"
+BUILDER_IMAGE="appdev-documentation-builder"
+BUILDER_CONT="appdev-documentation-builder-container"
+DEPLOY_IMAGE="appdev-documentation-deploy"
+TAG_LENGTH=7
+
+TARGET_DIR="html"
+
+function tag_push() {
+    TARGET_IMAGE=$1
+    USERNAME=$2
+    PASSWORD=$3
+
+    docker tag ${DEPLOY_IMAGE} ${TARGET_IMAGE}
+    if [ -n "${USERNAME}" ] && [ -n "${PASSWORD}" ]; then
+        docker login -u ${USERNAME} -p ${PASSWORD} -e noreply@redhat.com
+    fi
+    docker push ${TARGET_IMAGE}
+
+}
+
+# Exit on error
+set -e
+
+if [ -z $CICO_LOCAL ]; then
+    [ -f jenkins-env ] && cat jenkins-env | grep -e PASS -e GIT > inherit-env
```

Comment:
please add `-e DEVSHIFT` so that we get username and password from credentials sourced